gulpfile.js

```diff
@@ -91,6 +91,7 @@ function buildExtensions(options) {
   buildExtension('amp-user-notification', '0.1', true, options);
   buildExtension('amp-vimeo', '0.1', false, options);
   buildExtension('amp-vine', '0.1', false, options);
+  buildExtension('amp-google-vrview-image', '0.1', false, options);
```

Comment:
Yeah. We've been bikeshedding it a bit. /cc @cramforce 

Basically the decision we made is to start with a longer name and promote it to a shorter name based on feedback.